dces-drc-integration/src/integrationTest/java/uk/gov/justice/laa/crime/dces/integration/service/FdcIntegrationTest.java

```diff
@@ -313,4 +309,64 @@ void givenSomeNegativeFdcStatusTestWaitingItems_whenProcessDailyFilesRuns_thenTh
 			softly.assertThat(fdcContribution.getContFileId()).isNull();
 		});
 	}
+
+	/**
+	 * <h4>Scenario:</h4>
+	 * <p>A negative integration test to check that fdc_contributions having status REQUESTED are NOT updated to SENT if
+	 *    DRC processing responded synchronously with failure.</p>
+	 * <h4>Given:</h4>
+	 * <p>* 3 fdc_contributions record IDs that have been updated to the REQUESTED status.</p>
+	 * <h4>When</h4>
+	 * <p>* The {@link FdcService#processDailyFiles()} method is called.</p>
```

Comment:
I think the "When" part of the comment should also mention "and the call to DrcClient#sendFdcUpdate returns false".

Reply:
Yes, it should. Will change in next PR.